views/movies/new-movie.hbs

```diff
@@ -0,0 +1,26 @@
+<h2>Create Movie</h2>
+
+<form method = "post" action="/celebrities/create">
+
+    <label>Title:
+        <input type="text" name = "title">
+    </label>
+
+     <label>Genre:
+        <input type="text" name = "genre">
+    </label>
+
+     <label>Plot:
+        <input type="text" name = "plot">
+    </label>
+
+     <label>Cast:
+        <input type="text" name = "cast" multiple>
```

Comment:
You would want to print in the whole array of celebrities that would be possible to put as a cast in the movie

```suggestion
         <select name="cast" multiple>
            {{#each celebritiesArr}}
               <option value={{this._id}}> {{this.name}} </option>
            {{/each}}
        </select>
```